trunk/SUAVE/Components/Component.py

```diff
@@ -45,6 +45,9 @@ def __defaults__(self):
             """         
         self.tag    = 'Component'
         self.origin = [[0.0,0.0,0.0]]
+        self.max_per_vehicle = 0
+        self.PGM_characteristics = []

```

Comment:
If the PGM files will be stored separately I think we may want to remove defaulted values from the main repo.

Reply:
Let's talk tomorrow.